aws-common/src/main/java/io/druid/common/aws/ec2/EC2EnvironmentConfig.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package io.druid.indexing.overlord.autoscaling.ec2;
+package io.druid.common.aws.ec2;
```

Comment:
`AWSModule` needs to access the `getRegion` method on the `EC2EnvironmentConfig` class to set region appropriately -- I figured it was better to have a dependency from `indexing-service` to `aws-common` than from `aws-common` to `indexing-service`